browse/templates/home/news.html

```diff
@@ -6,17 +6,15 @@
 Celebrating <a href="https://blog.arxiv.org/2021/08/13/celebrating-arxivs-30th-anniversary/" target="_blank">arXiv's 30th anniversary!</a> 1991-2021.
 <br><br>
 {%- endif -%}
-{%- if rd_int >= 202110240100 and rd_int <= 202110312359 -%}
-Show your support for Open Science by <a href="https://bit.ly/arXivDONATEa" target="_blank">donating to arXiv</a> during International Open Access Week, October 25th-31st.
-<br><br>
-{%- endif -%}
 
-{%- if rd_int >= 202010010756 and rd_int <= 202011300100 -%}
-arXiv now processes new submissions and replacements with TeX Live 2020. <a href="https://blogs.cornell.edu/arxiv/2020/09/24/tex-live-2020-release-oct-1-2020/" target="_blank">Learn more</a>.
+<!-- annual post-giving-week message -->
+{%- if rd_int >= 202111010100 and rd_int <= 202111072359 -%}
+A big thank you for donating to arXiv during International Open Access Week, October 25th-31st. It's not too late to show your support. <a href="https://bit.ly/arXivDONATEa" target="_blank">Donate to arXiv here</a>.
```

Comment:
Suggestion: delete the dates, now that OA week is over

Reply:
done!